test/features/install_explainability.feature

```diff
@@ -0,0 +1,53 @@
+@explainability
+@kafka
+Feature: Kogito Data Index
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Kafka operator
```

Comment:
This line can be moved to `Install Kogito Explainability` scenario to Given part, as the other scenario use also Infinispan operator.